### DISCUSSION TITLE
ITF | Interleaved 2 of 5

### DISCUSSION THREAD

**Discussion by globalideias** [journeyapps/zxing-android-embedded#737] [General]
I have a 48-digit code, where for every 11 digits there is 1 digit that corresponds to the checksum, but the checksum digits are not read.

Only 44 digits are read.

**Comment by globalideias** [comment]
![cod-bar-pag](https://github.com/journeyapps/zxing-android-embedded/assets/10763719/723753e7-b3ee-4ad4-812c-4af15c2e0829)

**Reply by rkistner** [reply]
It looks like in this case, those checksum digits are only present in the printed text, and not in the barcode.

Since it's just checksums, you can calculate it after scanning.

**Reply by globalideias** [reply]
_"It looks like in this case, those checksum digits are only present in the printed text, and not in the barcode."- I too am coming to this conclusion.

So, I'm going to think of a solution, because I need the data to be validated before going to the api.

Thanks
